<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>From</class>
 <widget class="QWidget" name="From">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bgd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/hall_picture/bgd_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="person_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>980</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic; color:#aaff00;&quot;&gt;个人中心&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>120</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="release_buttton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>980</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic; color:#00ffff;&quot;&gt;发布信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushBtrron
{
background-color:rgb(170,170,255);
color:white;
border-radius:50px;
border:2px groove gray;
border-style:outset;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1030</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/hall_picture/返回.png</normaloff>:/hall_picture/返回.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>1051</width>
     <height>400</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>物品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>物品类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>丢失时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>丢失地点</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>联系方式</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>物品价值</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>-10</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>1</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#00007f;&quot;&gt;失物招领&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>失物启示</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>50</y>
     <width>1051</width>
     <height>461</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>物品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>物品类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发现时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发现地点</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>联系方式</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>存放地点</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>物品价值</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
